### ISSUE TITLE
Feature request: add presence for cqc

### ISSUE THREAD

**Issue by norohind** [SayakMukhopadhyay/EDMC-Discord-Presence#24]
There is no right way to see if player in cqc from game journals. But I can suggest a few indirect ways:
1. `LoadGame` event without `Ship` field happening on enter in cqc from main menu (can be used to detect enter in cqc)
2. `Music` event with `"MusicTrack":"MainMenu"` happening when exit from cqc to main menu (can be used to detect exit from cqc)

**Comment by norohind** [comment]
Now it is impossible because EDMC don't pass any events when you are in CQC, so we have to hope that `journal_entry_cqc()` [plugin function](https://github.com/EDCD/EDMarketConnector/issues/1030#issuecomment-835537759) will be implemented in EDMC

**Comment by jlabeeuw** [comment]
yes, using the music tracks is a way to check if the cqc is running:
entry['MusicTrack'] in ['CQC']:
==>Means, CQC is active (game running)
entry['MusicTrack'] in ['NoTrack','Exploration', 'Combat_Dogfight']:
==>Means: CQC is still active (no state change)
entry['MusicTrack'] in ['CQCMenu']
==>CQC menu.. queing up... 
and otherwise:
==> No CQC

**Comment by SayakMukhopadhyay** [comment]
Thanks for the feature request. I will need to test out the above methods and see what fits the best and if they are reliable I will add this feature

**Comment by norohind** [comment]
Since [`journal_entry_cqc`](https://github.com/EDCD/EDMarketConnector/pull/1232) EDMC functionality is merged to develop, we can expect it appearance in 5.2.0 EDMC version and then there will be no obstacles to implement this feature request. I already have modified plugin and going to PR it once EDMC 5.2.0 will be released.
